ow_lander/materials/scripts/ow_lander.frag

```diff
@@ -184,10 +186,18 @@ void lighting(vec3 wsDirToSun, vec3 wsDirToEye, vec3 wsNormal, vec4 wsDetailNorm
   const float specular_power = 100.0;
 
   // shadows
+  // Compute shadow lookup bias using formula from
+  // http://www.opengl-tutorial.org/intermediate-tutorials/tutorial-16-shadow-mapping/
+  // Should be able to bake this bias into the shadow map using constant_bias
+  // and slope_scale_bias in IRGShadowParametersPlugins, but it doesn't work.
+  float constantBias = vsPos.z * 0.00001 - 0.00002; // Normally a constant, but this works better with PSSM
```

Comment:
I have the same comment on this variable name as I did in the other PR.

Reply:
I debated this name too. In the end, I think the comment explains it well enough and the name constantBias will make it easier to reconcile with any shadow bias literature you read.